include/onnxruntime/core/graph/graph.h

```diff
@@ -1564,10 +1556,14 @@ class Graph {
 #if !defined(ORT_MINIMAL_BUILD)
   IOnnxRuntimeOpSchemaCollectionPtr schema_registry_;
 
-  // Container to hold initialized function bodies
-  std::vector<std::unique_ptr<onnxruntime::Function>> function_container_;
-
   std::unordered_map<std::string, const ONNX_NAMESPACE::FunctionProto*> model_local_functions_;
+
+  std::unordered_map<std::string, FunctionTemplate*> model_local_function_templates_;
+
```

Comment:
So, these pointers are owned by the Model class?  So it means a graph is owned by a model and once the graph is created, it can't be copied or transferred to another model.  Then why don't we just let graph hold a pointer of the model class?

Reply:
ok, there is already a reference to the model defined in graph.... update the implementation to use that reference.